tests/test_smoke.py

```diff
@@ -40,7 +40,6 @@ def test_all_apps():
             "pg-renamecluster",
             "pg-updatedicts",
             "pg-lsclusters",
-            "syncobj-admin",
```

Comment:
Since it's a python dep, it should tell us if the PYTHONPATH is not correct. We should most probably revisit this list and enable everything that has a working help command.